apps/learner/src/lib/components/Collection/Collection.svelte

```diff
@@ -52,7 +52,10 @@
       </div>
 
       <div class="flex flex-col">
-        <span class="text-sm">{numberofpodcasts} podcasts</span>
+        <span class="text-sm">
+          {numberofpodcasts}
```

Comment:
Is there a space between the number value and the podcast text?

Reply:
Added `&nbsp;` to allow readability.